src/frontends/android/app/src/main/java/org/strongswan/android/ui/adapter/VpnProfileAdapter.java

```diff
@@ -76,7 +77,7 @@ public View getView(int position, View convertView, ViewGroup parent)
 		tv = (TextView)vpnProfileView.findViewById(R.id.profile_item_certificate);
 		if (profile.getVpnType().has(VpnTypeFeature.CERTIFICATE))
 		{
-			tv.setText(getContext().getString(R.string.profile_user_certificate_label) + " " + profile.getUserCertificateAlias());
+			tv.setText(getContext().getString(R.string.profile_user_certificate_label) + " " + (TextUtils.isEmpty(profile.getUserCertificateAlias())?getContext().getString(R.string.certificate_absent):profile.getUserCertificateAlias()));
```

Comment:
This line is hardly readable. Maybe extract text to some string before and then set. Also check spaces between ? and :